server/routes/mcp/tokenizer.ts

```
@@ -0,0 +1,370 @@
+          return modelId.replace("x-ai/", "xai/");
+        }
+        if (modelId.startsWith("z-ai/")) {
+          return modelId.replace("z-ai/", "zai/");
+        }
+        // Already prefixed and doesn't need normalization, return as-is
+        return modelId;
+      }
+
+      // For models without prefix, look up provider and construct the string
+      const modelDef = getModelById(modelId);
+      if (modelDef) {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)